### PR TITLE
Add VGG trainer

### PR DESCRIPTION
**Pull request by galshachaf**
issue #62

### CODE REVIEW

File: trainer.py [hunk 1]
```diff
@@ -221,10 +224,8 @@ def __init__(self, layers_to_freeze=None, weight_map=None, **kwargs):
     def go(self):
         layers = self._create_layers()
         self.freeze_layers(layers, self._layers_to_freeze)
-        self.logger.debug("Freezing layers {}".format(self._layers_to_freeze))
         self.set_layers_weights(layers, self._weight_map)
         connected_layers = self._connect_layers(layers)
-        # TODO: Shouldn't layers[0] be connected_layers[0]?
```
Comment (dorimedini) [inline trainer.py:227]: I'll take that as a 'no' then :)

File: experiment.py [hunk 2]
```diff
@@ -233,19 +233,21 @@ def return_acc(acc: float, clean_acc: float, relative=False):
...
-                              "".format("contain" if no_new_layer_sets else "doesn't have",
-                                        [str(layers) for layers in layer_indices_list]))
-            self.logger.debug("Previous results {} all checkpoints in {}"
-                              "".format("contain" if no_new_epochs else "doesn't have",
-                                        [str(epoch) for epoch in checkpoint_epochs]))
-            if no_new_layer_sets and no_new_epochs:
-                self.logger.debug("Nothing new to compute, returning previous results")
-                return prev_results, prev_clean
+            if prev_results:
+                print("shouldn't get here")
```
Comment (dorimedini) [inline experiment.py]: Why shouldn't we get here...? If we successfully loaded results from disk we should be here

**Review by dorimedini — CHANGES_REQUESTED** [review]

**Comment by galshachaf** [comment]
whoops, that's a debug statement gone wrong

**Review by dorimedini — APPROVED** [review]

File: experiment.py (hunk 2 above)
Comment (galshachaf) [reply experiment.py]: fixed in future commit

**Pull request merged**